web/src/infra/gameInfo/GameInfo.tsx

```diff
@@ -96,4 +95,4 @@ class GameInfo extends React.Component<gameInfoProps, {}> {
   }
 }
 
-export default withRouter(GameInfo as any);
```

Comment:
Important to mention that no properties from Router were been used on this component, so I decided to remove the HOC.